### DISCUSSION TITLE
Automatic backup

### DISCUSSION THREAD

**Discussion by Spiny150** [leonlatsch/Photok#361] [Ideas]
**The Idea :**
Add an option to automatically create backups at a selected time interval.

**Why it's useful :**
Users could then sync their files to their preferred cloud storage space, avoiding data loss if the user breaks or loses their phone.

**Additional note :**
Older backups would probably have to be deleted for reasons of storage constraints.

**Comment by leonlatsch** [comment]
Hey,

Something along these lines is already considered in #354

Also #197 plans to have the possibility to store everything in an external location (cloud) aswell.
